### DISCUSSION TITLE
There is a problem accessing project 02

### DISCUSSION THREAD

**Discussion by Zeyad-hs** [cooperrc/computational-mechanics#245] [Q&A] [answered]
Every time I try to access the project I get the message shown in the attached photo!
![Project_02_problem ](https://github.com/cooperrc/computational-mechanics/assets/170972675/b6b20f2f-a3dd-4b55-99a9-867c6913157c)

**Comment by Kevin-Harmeling** [comment]
I am having the same issue.

**Comment by tzamani310** [comment]
Same here!

**Comment by cooperrc** [comment]
Oh no! 
I'm look into the issue now.

**Comment by cooperrc** [comment, marked as answer]
[Video solution - youtube channel](https://youtu.be/92HVSPjDfDs)

There's a typo in the link:

```markdown
## [Project #02 - NYSE random walk predictor](../projects/02_Analyze-data_project)
```

should be 

```markdown
## [Project #02 - NYSE random walk predictor](../projects/02_Analyze-data_project.md)
```

You can double-click the cell and and the `.md` to the end of the file location, **sorry!!**
